superset-frontend/src/explore/components/controls/FixedOrMetricControl/index.jsx

```diff
@@ -127,7 +127,7 @@ export default class FixedOrMetricControl extends React.Component {
           <Collapse.Panel
             showArrow={false}
             header={
-              <Label onClick={() => undefined}>
+              <Label role="button" onClick={() => undefined}>
```

Comment:
Since onClick handler does nothing, both role and onClick are redundant.

```suggestion
              <Label>
```